src/bpftrace.cpp

```diff
@@ -1175,6 +1178,13 @@ int BPFtrace::run(BpfBytecode bytecode)
   if (std::getenv("__BPFTRACE_NOTIFY_PROBES_ATTACHED"))
     std::cerr << "__BPFTRACE_NOTIFY_PROBES_ATTACHED" << std::endl;
 
+#ifdef HAVE_LIBSYSTEMD
+  err = sd_notify(false, "READY=1\nSTATUS=Processing events...");
```

Comment:
Should the first argument be `0` instead? The docs seem to say it takes an integer:
> `int unset_environment`

Reply:
The API uses int because `bool` was not a thing in C before `C99`. Every call to this function in the systemd codebase uses `true/false` so I did the same here.